_includes/tutorials/aviation/confluent/code/tutorial-steps/dev/process.sql

```diff
@@ -1,90 +1,95 @@
 SET 'auto.offset.reset' = 'earliest';
 
-CREATE TABLE customers (ID             INT     PRIMARY KEY
-                       , NAME           VARCHAR
-                       , ADDRESS        VARCHAR
-                       , EMAIL          VARCHAR
-                       , PHONE          VARCHAR
-                       , LOYALTY_STATUS VARCHAR)
-              WITH (KAFKA_TOPIC = 'customers'
-                   , FORMAT = 'JSON'
-                   , PARTITIONS = 6
+CREATE TABLE customers (
+  ID INT PRIMARY KEY,
+  NAME VARCHAR,
+  ADDRESS VARCHAR,
+  EMAIL VARCHAR,
+  PHONE VARCHAR,
+  LOYALTY_STATUS VARCHAR
+) WITH (
+  KAFKA_TOPIC='customers',
+  FORMAT='JSON',
+  PARTITIONS=6
 );
 
-CREATE TABLE flights (ID               INT     PRIMARY KEY
-                       , ORIGIN        VARCHAR
-                       , DESTINATION   VARCHAR
-                       , CODE          VARCHAR
-                       , SCHEDULED_DEP TIMESTAMP
-                       , SCHEDULED_ARR TIMESTAMP)
-              WITH (KAFKA_TOPIC = 'flights'
-                   , FORMAT = 'JSON'
-                   , PARTITIONS = 6
+CREATE TABLE flights (
+  ID INT PRIMARY KEY,
+  ORIGIN VARCHAR,
+  DESTINATION VARCHAR,
+  CODE VARCHAR,
+  SCHEDULED_DEP TIMESTAMP,
+  SCHEDULED_ARR TIMESTAMP
+) WITH (
+  KAFKA_TOPIC='flights',
+  FORMAT='JSON',
+  PARTITIONS=6
 );
 
-CREATE TABLE bookings (ID            INT     PRIMARY KEY
-                       , CUSTOMER_ID INT
-                       , FLIGHT_ID   INT)
-              WITH (KAFKA_TOPIC = 'bookings'
-                   , FORMAT = 'JSON'
-                   , PARTITIONS = 6
+CREATE TABLE bookings (
+  ID INT PRIMARY KEY,
+  CUSTOMER_ID INT,
+  FLIGHT_ID INT
+) WITH (
+  KAFKA_TOPIC='bookings',
+  FORMAT='JSON',
+  PARTITIONS=6
 );
 
 CREATE TABLE customer_bookings AS 
   SELECT C.*, B.ID, B.FLIGHT_ID
-  FROM   bookings B
-          INNER JOIN customers C
-              ON B.CUSTOMER_ID = C.ID;
+  FROM bookings B
+  INNER JOIN customers C
+  ON B.CUSTOMER_ID = C.ID;
```

Comment:
Something to consider and just my personal opinion:  it doesn't really help the reader to abbreviate table and stream names like this. `bookings` and `customers` aren't long complicated names and `B` and `C` aren't very helpful abbreviated names in their own right.

Reply:
Let's leave as-is for now.